sinopia_client/test/api/LDPApi.spec.js

```diff
@@ -117,6 +117,10 @@
             .catch(function(err) { console.error(`Error adding ${opts['slug']}: ${err}`) })
         });
       });
+
+      describe('with an RDF resource', function() {
+        // TODO: a test: https://github.com/LD4P/sinopia_server/issues/34
```

Comment:
rando question:  would it make sense to add "skip" to the tests we're skipping, to avoid false positives in our output?

`describe.skip('with an RDF resource', function() {`

Reply:
good call, see #87